executor/src/witgen/global_constraints.rs

```diff
@@ -380,7 +380,7 @@ mod test {
 
     #[test]
     fn zero_one() {
-        let fixed = [0, 1, 0, 1, 0].map(|v| v.into());
+        let fixed = [0, 1, 0, 1].map(|v| v.into());
```

Comment:
The old test is not working anymore, but I think it is also an unrealistic test, because in practice fixed columns will always have a size which is a power of 2.